manifests/init.pp

```diff
@@ -139,6 +142,8 @@
   Integer[0] $pulpcore_api_service_worker_timeout = 90,
   Boolean $pulpcore_cache_enabled = true,
   Optional[Variant[Integer[1], Enum['None']]] $pulpcore_cache_expires_ttl = undef,
+  Variant[Stdlib::Absolutepath, Array[Stdlib::Absolutepath]] $additional_pulpcore_import_paths = [],
+  Variant[Stdlib::Absolutepath, Array[Stdlib::Absolutepath]] $additional_pulpcore_export_paths = [],
```

Comment:
Since both of these are *plural* I would expect it to only allow array.

Reply:
While this is true, passing arrays in the installer is cumbersome and allowing strings here (and convert them later) makes the UX better, IMHO

Reply:
> While this is true, passing arrays in the installer is cumbersome and allowing strings here (and convert them later) makes the UX better, IMHO

This was my reasoning. I kept the option to pass a string for now, let me know if there is other feedback on it.